libs/core/__deprecated__/renderer/components/rest/menu/menu-bottom.component.tsx

```diff
@@ -51,14 +52,52 @@ interface Props {
 const MenuBottom: FunctionComponent<Props> = ({ dataSyncInProgress }) => {
   const history = useHistory()
   const devices = useSelector(getDevicesSelector)
-  const deviceInitializationStatus = useSelector(getDeviceInitializationStatus)
-  const deviceInitialized =
-    deviceInitializationStatus === DeviceInitializationStatus.Initialized
+  const dispatch = useDispatch<Dispatch>()
 
   const handleSelectDeviceClick = () => {
     history.push(URL_DISCOVERY_DEVICE.availableDeviceListModal)
   }
 
+  const isSelectDevice =
+    !dataSyncInProgress &&
+    devices.length > 0 &&
+    history.location.pathname === URL_MAIN.news
+
+  const SelectDeviceButton = (
+    <DeviceButtonWrapper>
+      <DeviceButton
+        label={intl.formatMessage({
+          id: "component.deviceSelection.selectDevice",
+        })}
+        displayStyle={DisplayStyle.BorderlessButton}
+        Icon={IconType.DotsInBox}
+        iconBadgeCountIndicator={devices.length}
+        onClick={handleSelectDeviceClick}
+      />
+    </DeviceButtonWrapper>
+  )
+
+  const isChangeDevice =
+    !dataSyncInProgress &&
+    devices.length > 1 &&
+    history.location.pathname !== URL_MAIN.news
+
+  const ChangeDeviceButton = (
```

Comment:
🟢 
I'm considering moving the `SelectDeviceButton` and `ChangeDeviceButton` components to separate files. This would allow for:

1. **Modularity:** Having each component in a separate file would facilitate code organization and development.
2. **Clearer Project Structure:** Separate files would make navigation and understanding of the component hierarchy in the application easier.

Reply:
Not sure about that, SelectDeviceButton and ChangeDeviceButton are only implementations of DeviceButton. I am just passing props to that component, nothing more.

I've moved it out of return to be more readable.

Reply:
Moving this outside the main component block could be a compromise for me. However, as I mentioned at the beginning of the comment, please don't get blocked by my remark, as it is not blocking at all.

Reply:
Moving this outside the main component block could be a compromise for me. However, as I mentioned at the beginning of the comment, please don't get blocked by my remark, as it is not blocking at all.